spec/lib/admin_user_index_query_spec.rb

```diff
@@ -80,10 +80,12 @@ def real_users(query)
     TrustLevel.levels.each do |key, value|
       it "finds user with trust #{key}" do
         user = Fabricate(:user, trust_level: value)
+
+        next if value + 1 == 5
```

Comment:
Might be clearer if we used the enum rather than 5?

Reply:
Nice didn't know that was a thing, will fix.